bundles/org.openhab.binding.deconz/src/main/resources/OH-INF/thing/sensor-thing-types.xml

```diff
@@ -544,12 +559,26 @@
 		<config-description-ref uri="thing-type:deconz:sensor"/>
 	</thing-type>
 
+	<channel-type id="locked">
+		<item-type>Switch</item-type>
+		<label>Locked</label>
+		<description>Status of this thermostat's childlock.</description>
+		<category>Lock</category>
+	</channel-type>
+	<channel-type id="windowopen">
+		<item-type>Contact</item-type>
+		<label>Window Open</label>
+	</channel-type>
+	<channel-type id="externalwindowopen">
+		<item-type>Contact</item-type>
+		<label>External Window Open</label>
```

Comment:
Would a description be helpful?

Reply:
It would, it I knew exactly what this one is doing :-)